builder/builder_test.go

```diff
@@ -0,0 +1,51 @@
+package builder
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func mustExist(t *testing.T, name string) {
+	t.Helper()
+	_, err := os.Stat(name)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func cleanup(path string) {
+	_ = os.RemoveAll(path)
+}
+
+func TestNewDNFConfOnInit(t *testing.T) {
+	testDir, err := ioutil.TempDir("", "dnf-testdir-")
+	if err != nil {
+		t.Fatalf("couldn't create temporary directory to write test cases: %s", err)
+	}
+	defer cleanup(testDir)
+	conf := testDir + "/builder.conf"
+
+	b := New()
+	b.Config.LoadDefaultsForPath(false, testDir)
```

Comment:
The signature of this function is `func (config *MixConfig) LoadDefaultsForPath(path string)`, so I think the lint warnings here do make sense...

Reply:
Ok rebased and now this is fixed...this was a weird scenario...